client/internal/routemanager/client.go

```diff
@@ -140,6 +149,39 @@ func (c *clientNetwork) getBestRouteFromStatuses(routePeerStatuses map[string]ro
 	return chosen
 }
 
+// calculateLatencyScores calculates the latency scores for route based on the latency of the peers.
+// The output will return a map of peer ID to latency score. The lower the latency, the lower the score.
+// The highest latency will have a score of 1, the second highest will have a score of 2, and so on.
+// The lowest latency will have the highest score.
+func calculateLatencyScores(routePeerStatuses map[string]routerPeerStatus) map[string]int {
```

Comment:
It might be better to work with a float score and convert the latency to float based on milliseconds.

This will prevent unforeseen cases like a  relayed connection will have lower latency than P2P

We still want P2P as it will provide top bandwidth

Reply:
e.g., 3 routing peers:

peer 1 with direct connection and latency of 0.29ms
peer 2 with a relayed remote connection, no proxy, and a latency of 0.26ms
peer 3 with local relayed connection and latency of 0.25ms


scores without account latency would be:

peer 1: 2
peer 2: 1
peer 3 0

with latency: 
peer 1: 3
pee2: 3
peer3: 3

the decision will be then made by choosing the first evaluated peer.

Reply:
Yes I see your point I will do a rework of that calculation